Makefile.in

```diff
@@ -367,3 +367,6 @@ doxygen:
 doxygen-upload:
 	rsync -avzv $(srcdir)/dox/html/ --delete \
 	$${SAMBA_HOST-samba.org}:/home/httpd/html/rsync/doxygen/head/
+
+sockopts.c: sockopts.c.sh
+	sh $< >$@.tmp && mv $@.tmp $@
```

Comment:
This is where your problem with FreeBSD is — `$<` is a GNU make extension, not a standard one† — you can see in the output how this is being executed:

```
sh  >sockopts.c.tmp && mv sockopts.c.tmp sockopts.c
```

The rest of `Makefile.in` appears to avoid this variable and would then use `$(srcdir)/sockopts.c.sh` as a full reference.

† Technically, POSIX make supports `$<` for `.DEFAULT` rules only: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/make.html

Ironically, it looks like the [FreeBSD man page](https://man.freebsd.org/cgi/man.cgi?make(1)) suggests otherwise:

> The `$@` and `$<` variables are more or less universally portable

Reply:
Thank you so much!